### DISCUSSION TITLE
My hometown pull request cannot be merged. Attached image for reference.

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#79] [Q&A]
@eculler 

![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/38798140/2a34280f-8de5-4e3e-a2c5-c0c04c0ed368)

**Comment by nquarder** [comment]
yes @DharaniSureshBabu that is because we have the settings such that only @eculler and I are able to merge the pull requests for the home-towns repository. We will merge soon! :)
